scenarios/kubernetes_e2e.py

```diff
@@ -334,6 +356,13 @@ def main(args):
     # So if you overwrite FOO=c for a local run it will take precedence.
     #
 
+    if (args.image_family or args.image_project) and args.mode == 'docker':
+        raise ValueError(
+            '--image-family / --image-project is not supported in docker mode')
+    if bool(args.image_family) != bool(args.image_project):
+        raise ValueError(
+            '--image-family and --image-project must be both set or unset')
```

Comment:
you can delete those now, or if you want to keep them, maybe move to `parse_args`